packages/scaffold-ui/src/views/w3m-connecting-wc-view/index.ts

```diff
@@ -80,13 +67,9 @@ export class W3mConnectingWcView extends LitElement {
         properties: { message: (error as BaseError)?.message ?? 'Unknown' }
       })
       ConnectionController.setWcError(true)
-      if (CoreHelperUtil.isAllowedRetry(this.lastRetry)) {
-        SnackController.showError((error as BaseError).message ?? 'Declined')
-        this.lastRetry = Date.now()
-        this.initializeConnection(true)
-      } else {
-        SnackController.showError((error as BaseError).message ?? 'Connection error')
-      }
+      SnackController.showError((error as BaseError).message ?? 'Connection error')
```

Comment:
this is no longer looped into reconnecting right? Just triple checking there won't be popups going crazy if a network issue happens and we enter a retry loop

Reply:
nope, when `await ConnectionController.connectWalletConnect()` rejects for whatever reason 
- rejected by the user
- proposal expired
the router redirects back to wallet list and thats all